std_msgs/Header header

float64 yaw_rate    # Yaw rate as reported in rad/s.
